prototypes/signal.lua

```diff
@@ -0,0 +1,125 @@
+local capture_tint = {0.9,0.9,0.9}
+local target_tint = {0.1,0.6,0.2}
+
+data:extend({
+    {
+        type = "item-subgroup",
+        name = "virtual-signal-TLBE",
+        group = "signals",
+        order = "z",
+    },
+    {
+        type = "virtual-signal",
+        name = "signal-capture-north-west",
+        icons = {
+            {
+                icon = "__TLBE__/graphics/box-north-west.png",
+                tint = capture_tint
+            }
+        },
+        icon_size = 32,
```

Comment:
I feel the tags as a little small on the map. It looks clean right now, but I find it hard to see them (especially the white ones)

Reply:
I'm not a gui person - I'll go with whatever you suggest for look and feel.